exit.go

```diff
@@ -30,10 +49,13 @@ func ExitReload(prefix string, reload func(), breakdown func()) {
 	// Wait for exit signal
 	signal.Notify(sc, syscall.SIGHUP, syscall.SIGINT, syscall.SIGTERM, os.Interrupt, os.Kill)
 	<-done
+	isRunning = false
```

Comment:
why is this variable modified before locking the mutex?

Reply:
This doesn't matter as it should already be attempting to breakdown; therefore any active reloads should just return when possible.
I can, however, move it below the mutex lock/unlock.